dask/array/utils.py

```diff
@@ -96,10 +96,8 @@ def assert_eq(a, b, check_shape=True, **kwargs):
                              os.linesep.join(diff))
 
     try:
-        if _not_empty(a) and _not_empty(b):
-            # Treat all empty arrays as equivalent
-            assert a.shape == b.shape
-            assert allclose(a, b, **kwargs)
+        assert a.shape == b.shape
+        assert allclose(a, b, **kwargs)
         return
```

Comment:
Was this suppose to return something (not `None`)?

Reply:
Maybe `True` like the `return` below?

Reply:
Sure, I'll return `True` to match the below.